packages/choreography-lint/README.md

```diff
@@ -0,0 +1,2 @@
+# Choreography-ts ESLint Plugin
+This package defines and exports custom ESLint linting rules for choreography-ts. The [custom rules](https://eslint.org/docs/latest/extend/custom-rules) are defined in separate files under the `rules` directory, are exported with the corresponding ID's in the `rules/rules.js` file, and are then configured in the `dist/index.js` main [plugin](https://eslint.org/docs/latest/extend/plugins) file which specifies the error levels for the rules. To activate the rules, the plugin must be extended in the project's main `.eslintrc.*` [configuration file](https://eslint.org/docs/latest/use/configure/configuration-files) after this package is installed by running `pnpm install` at the root of the project. [Unit tests](https://eslint.org/docs/latest/extend/plugins#testing) for the custom rules are implemented under the `tests` directory.
```

Comment:
I want you to configure the package with https://typescript-eslint.io/ so we'll have access to type information. It might not be necessary for this particular rule but we'll definitely need it in future. It should also makes it easy to write rules and tests in TypeScript, not JavaScript.